test/subarray.jl

```diff
@@ -190,7 +190,7 @@ function runtests(A::Array, I...)
     test_bounds(S)
 end
 
-function runtests(A::ANY, I...)
```

Comment:
yeah there might be something fishy here, you can check with `make testall1` - though maybe comment out a bunch of the non-subarray tests from choosetests.jl to make it go faster

here's the error I'm getting, when trying to call `runtests("compile")`
```
ERROR: LoadError: BoundsError: attempt to access ()
  at index [0]
 [inlined code] from .\tuple.jl:8
 in runtests(::Any) at C:\cygwin64\home\Tony\julia\test\subarray.jl:202
 in (::##11#19)() at C:\cygwin64\home\Tony\julia\test\runtests.jl:69
 in cd(::##11#19, ::ASCIIString) at .\file.jl:58
 in include(::ASCIIString) at .\boot.jl:233
 in include_from_node1(::UTF8String) at .\loading.jl:426
 in process_options(::Base.JLOptions) at .\client.jl:263
 in _start() at .\client.jl:319
while loading C:\cygwin64\home\Tony\julia\test\runtests.jl, in expression starting on line 13
```